network-admin.php

```diff
@@ -5,7 +5,7 @@
  * @package Mercator
  */
 
-namespace Mercator\Admin;
+namespace Mercator\GUI;
 
```

Comment:
`use Mercator\Admin\Alias_List_Table;`
needs to be added otherwise there is no access to the class